server/routerlicious/packages/services/src/sessionTracker.ts

```diff
@@ -104,7 +104,7 @@ export class CollaborationSessionTracker implements ICollaborationSessionTracker
 			lastClientLeaveTime: undefined,
 			telemetryProperties: {
 				hadWriteClient:
-					existingSession?.telemetryProperties?.hadWriteClient || client.isWriteClient,
+					existingSession?.telemetryProperties?.hadWriteClient ?? client.isWriteClient,
```

Comment:
Was it flagged and auto-fixed by the linter? If so, I think it might have a bug handling `boolean | undefined`, the previous change would result in runtime behavior changes on `false || true`.
```suggestion
					(existingSession?.telemetryProperties?.hadWriteClient ?? false) || client.isWriteClient,
```

Reply:
Oof. I don't remember if this was auto-fixed or manual. Good catch!